### DISCUSSION TITLE
Update expat to 2.6.3 and libwebp to 1.4.0

### DISCUSSION THREAD

**Discussion by jeremy-bridges** [mono/SkiaSharp#3053] [Ideas]
Many vulnerabilities are flagged by Black Duck for these two libraries. Please update them to reduce the security vulnerability profile of SkiaSharp.

expat:
![image](https://github.com/user-attachments/assets/0089fbc7-a288-4a42-a275-a00ecc1df86c)

libwebp:
![image](https://github.com/user-attachments/assets/063fceb6-d0d0-4e53-81e1-7d256f594102)

**Comment by mattleibow** [comment]
We got this tracked in #3055. Thanks for letting us know.

The libexpat landed and is on nuget already.
